CHANGELOG.md

```diff
@@ -137,6 +137,7 @@
 
 # 0.33.0
 * Breaking changes:
+  * Dropped support for Ruby versions older than 2.4.
```

Comment:
I don't know that I would list this as a breaking change. We haven't considered this to be breaking for other gems.

Reply:
How about listing it under a new *Backwards incompatible changes* section? While it isn't necessarily a breaking change, it's still a pretty big change in support.

Reply:
We have used "Other" to describe this change in the Google Cloud Platform gems.

https://github.com/googleapis/google-cloud-ruby/releases/tag/stackdriver/v0.16.0

This isn't an API change like most of the changes listed.

Reply:
That works for me. 😄